Graphics/Win32/Window.hsc

```diff
@@ -228,6 +242,21 @@ foreign import WINDOWS_CCONV unsafe "windows.h SetWindowLongPtrW"
 #endif
   c_SetWindowLongPtr :: HWND -> INT -> Ptr LONG -> IO (Ptr LONG)
 
+#if defined(x86_64_HOST_ARCH)
+foreign import WINDOWS_CCONV "windows.h GetWindowLongPtrW"
+  c_GetWindowLongPtr :: HANDLE -> INT -> IO LONG_PTR
+#elif defined(x86_64_HOST_ARCH)
```

Comment:
The CI is dying on this, which needs to be `i386_HOST_ARCH`. odd that the error wasn't shown on appveyor..